uaclient/cli.py

```diff
@@ -320,6 +323,20 @@ def print_version(_args=None, _cfg=None):
     print(version.get_version())
 
 
+def action_refresh(args, cfg):
+    if not cfg.is_attached:
+        print(ua_status.MESSAGE_UNATTACHED)
+        return 1
+    contract_token = cfg.read_cache('contract-token')
+    if contract_token:
+        contract_token = contract_token['contractToken']
+    if contract.request_contract_updates(cfg, contract_token=contract_token):
+        print('Refreshed Ubuntu Advantage contracts.')
```

Comment:
Should this string be in `ua_status` instead?

Reply:
I presume you're thinking about internationalization of messages... We probably should keep all printed statements, errors etc in one place. We'll probably have to separate thes out of ua/status.py as it's going to get fairly big I presume.